app/js/app/services/utils.js

```diff
@@ -0,0 +1,212 @@
+angular.module('app.services').factory('youtube',function () {
```

Comment:
@austinpapp We will want to eventually redesign the question template UI so that the video/picture/etc attached is not buried the More Tab. We will also want videos viewable from newsfeed directly